packages/landing/deploy.js

```diff
@@ -20,12 +20,34 @@ async function getHash(Bucket, Key) {
     return null;
 }
 
+/** If $GOOGLE_ANALYTICS is set, insert the tracking code into the index.html */
+async function insertGoogleAnalytics() {
+    const trackingId = process.env['GOOGLE_ANALYTICS'];
+    if (trackingId == null || trackingId.trim() == '') return;
+
+    const rawIndexHtml = await fs.readFile('./static/index.html');
+    const trackingCode = `<script async src="https://www.googletagmanager.com/gtag/js?id=${trackingId}"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', '${trackingId}');
+    </script>
+</html>`;
+
+    // Put the tracking at the bottom of the index.html
+    const outputHtml = rawIndexHtml.toString().replace('</html>', trackingCode);
+    await fs.writeFile('./static/index.html', outputHtml);
```

Comment:
A script tag can't go directly into an html tag

Reply:
I mean it "can" and it works in both chrome and firefox, ideally i guess it should live inside the head tag with a "defer"